hnn_core/gui/gui.py

```diff
@@ -1328,25 +1092,22 @@ def load_drive_and_connectivity(params, log_out, drives_out,
 
 def on_upload_change(change, params, tstop, dt, log_out, drive_boxes,
                      drive_widgets, drives_out, connectivity_out,
-                     connectivity_sliders, load_info, layout):
+                     connectivity_sliders, layout):
     if len(change['owner'].value) == 0:
         return
+    logger.debug("received new uploaded params file...")
+    # for compability
```

Comment:
Is this actually necessary now that that the `ipywidgets` is restricted in `setup.py`?